server.js

```diff
@@ -10,6 +10,12 @@ app.use(cors());
 /* --------------------------------------------------------------------- Routes */
 
 app.use("/auth", require("./routes/auth"));
+
+app.use("/", require("./routes/projets"));
+app.use("/", require("./routes/logos"));
+app.use("/", require("./routes/docFavart"));
+app.use("/", require("./routes/docExt"));
```

Comment:
Nope, that's not how you work with routers, you should have something like this:
```
app.use("/projets", require("./routes/projets"));
￼app.use("/logos", require("./routes/logos"));
￼app.use("/resources", require("./routes/docFavart"));
￼app.use("/resourcesext", require("./routes/docExt"));
```